src/app/component/comments/comments/add-comment/add-comment.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="wrapper">
+    <img src="../../../../assets/img/comment-avatar.png" alt="avatar">
+    <textarea name="" id="" placeholder="Add a comment"></textarea>
+    <button>Comment</button>
+</div>
```

Comment:
please add new line in the end of div tag

Reply:
fixed